src/view/placeholder.js

```diff
@@ -140,21 +140,21 @@ export function hidePlaceholder( writer, element ) {
  * {@link module:engine/view/placeholder~enablePlaceholder `enablePlaceholder()`} in that case or make
  * sure the correct element is passed to the helper.
  *
- * @param {module:engine/view/element~Element} element
+ * @param {module:engine/view/element~Element|module:engine/view/documentfragment~DocumentFragment} elementOrDocumentFragment
  * @returns {Boolean}
  */
-export function needsPlaceholder( element ) {
-	const doc = element.document;
-
-	// The element was removed from document.
-	if ( !doc ) {
+export function needsPlaceholder( elementOrDocumentFragment ) {
```

Comment:
Why those changes?

Reply:
I can see that we should use here `isAttached()` and that we don't need the type change.